schema/F99.json

```diff
@@ -340,6 +340,25 @@
                 "RULE_REFERENCE": null,
                 "FIELD_FORM_ASSOCIATION": null
             }
+        },
```

Comment:
It looks like the COL_SEQ of this file are wrong after 2.  Committee Name and everything after it should be -1

Reply:
Fixed